op-node/rollup/engine/events.go

```diff
@@ -266,6 +266,8 @@ func (d *EngDeriver) AttachEmitter(em event.Emitter) {
 	d.emitter = em
 }
 
+const blockTimeThreshold uint64 = 2
```

Comment:
I believe we can pull this from the [config](https://github.com/ethereum-optimism/optimism/blob/289f0748a8a23ddb865fe48fe9396a3fcc534d43/op-node/rollup/types.go#L70), not all op-stack chains have two second blocktimes.